### ISSUE TITLE
workbench.action.terminal.runSelectedText now changes focus

### ISSUE THREAD

**Issue by LarsWH** [microsoft/vscode#174822]
Type: <b>Bug</b>

For years I have enjoyed using this feature multiple times a day
(workbench.action.terminal.runSelectedText) and I always have a keyboard shortcut configured to run it for all sorts of commands (cmd, wsl, az cli, ...)

With one of the very latest updates, you have decided to change focus when running the command, and that completely made it useless to me. Very sad!!!

Others have observed this too:
https://www.reddit.com/r/vscode/comments/10zokkf/recent_update_changes_focus_when_executing/

/Lars

VS Code version: Code 1.75.1 (441438abd1ac652551dbe4d408dfcec8a499b8bf, 2023-02-08T21:32:34.589Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz (12 x 2592)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|31.76GB (13.13GB free)|
|Process Argv|C:\\Users\\lwh\\OneDrive - GateHouse Group\\Desktop\\NbGit.azcli --crash-reporter-id 32fd5b99-d982-4158-9f7a-1c249a8c1088|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (17)</summary>

Extension|Author (truncated)|Version
---|---|---
doxdocgen|csc|1.4.0
better-cpp-syntax|jef|1.17.2
vscode-graphviz|joa|0.0.6
cmake-language-support-vscode|jos|0.0.9
vscode-docker|ms-|1.23.3
vscode-dotnet-runtime|ms-|1.6.0
remote-containers|ms-|0.275.1
remote-ssh|ms-|0.96.0
remote-ssh-edit|ms-|0.84.0
remote-wsl|ms-|0.75.3
azurecli|ms-|0.5.0
cmake-tools|ms-|1.13.45
cpptools|ms-|1.14.3
cpptools-extension-pack|ms-|1.3.0
remote-explorer|ms-|0.2.0
vscode-print|pdc|0.10.20
cmake|twx|0.0.17

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vscaat:30438848
vsclangdf:30486550
c4g48928:30535728
dsvsc012cf:30540253
azure-dev_surveyone:30548225
pyindex848:30662994
nodejswelcome1:30587005
3biah626:30602489
pyind779:30662992
89544117:30613380
pythonsymbol12:30657548

```

</details>

**Comment by meganrogge** [comment]
I believe this is already fixed, can you see if you can reproduce in the [Insiders build](https://code.visualstudio.com/insiders)?
/label info-needed

**Comment by LarsWH** [comment]
Yes, it seems to be fixed in 1.76.0-insider
